@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1695,7 +1707,9 @@ export class TextElementsRenderer {
                     }
 
                     renderParams.fadeAnimationRunning =
-                        renderParams.fadeAnimationRunning || textIsFadingOut || textFading;
+                        renderParams.fadeAnimationRunning ||
+                        textRenderState!.isFadingOut() ||
```

Comment:
is there a reason to not store isFadingOut in the boolean, as it was?

Reply:
since we're changing textRenderState all along this function, it's confusing and error prone to store isFadingOut into a variable. We might change textRenderState after the variable is set. I'll change the function so that we don't update the state all over the place, but anyway it's still safer to get the state directly.